src/web/pages/portlists/PortRangesTable.tsx

```diff
@@ -45,16 +47,16 @@ const PortRangesTable: React.FC<PortRangesTableProps> = ({
       </TableHeader>
       <TableBody>
         {portRanges.map(range => (
-          <TableRow key={range.start + range.protocol_type}>
+          <TableRow key={`{range.start}-{range.protocolType}`}>
```

Comment:
Q: should it be interpolated?
```suggestion
          <TableRow key={`${range.start}-${range.protocolType}`}>
```